build.rs

```diff
@@ -52,6 +52,11 @@ type Date = &'static str;
 /// versions for which to enable particular features.
 type ConditionalCfg = (Cfg, &'static [(Date, Cfg)]);
 const CONDITIONAL_CFGS: &'static [ConditionalCfg] = &[
+	(None, &[("2019-02-24", Some("pattern_guards"))]),
+	(None, &[("2018-08-14", Some("non_exhaustive"))]),
+	(Some("collections"), &[("2018-08-13", None)]),
```

Comment:
Isn't this a dupe of `no_collections`?

Reply:
Good point! Yeah, I think `not(no_collections)` will do it. Testing this...

Reply:
Well, I cannot use the oldest versions of rust-nightly on macOS; I let `nightly-2016-03-11` run all night and it never finished. I'm running the test again starting with `nightly-2016-12-05`.

Reply:
AH! Yep, it works.